install/install.go

```diff
@@ -283,7 +330,35 @@ func K8sInstaller(c *k8s.Client, o Options) error {
 		if err != nil {
 			return err
 		}
-		fmt.Printf("KubeArmor manifest file saved to \033[1m%s\033[0m\n", f.Name())
+		s3:=f.Name()
+		progbar.filled=48
+		progbar.unfilled=0
+		progbar.done=(float64(progbar.filled)/progbar.size)*float64(100)
+		s:=""
+		for i:=0;i<progbar.filled;i++{
+			s=s+progbar.fill
+		}
+		progbar.bar=s
+		printMessage("🤩  KubeArmor manifest file saved to \033[1m"+s3+"\033[0m",false)
+
+	}
+	fmt.Printf("Checking if Karmor containers are running ...")
```

Comment:
fmt.Printf("Checking if KubeArmor pods are running...")

Reply:
okay will make the changes